ament_cmake_export_libraries/cmake/environment/library_path.bat.in

```diff
@@ -31,4 +29,3 @@ goto:eof
     set "%~1=%_list%"
   )
 goto:eof
-
```

Comment:
Like I said, if you're going to do the other changes, feel free to go ahead and merge this one. But the other pr wasn't around when you put this one in review, so I pointed out there were related, but missing, changes like this one and that this change was unrelated to the other changes in this pr.